ee/clickhouse/queries/funnels/test/test_funnel_persons.py

```diff
@@ -27,9 +27,97 @@ def _create_event(**kwargs):
 
 
 class TestFunnel(ClickhouseTestMixin, APIBaseTest):
-    def setUp(self):
-        self._create_sample_data()
-        super().setUp()
+    def _create_sample_data_multiple_dropoffs(self):
```

Comment:
Can we make this distribution non-symmetric? Such that `funnel_step = 2` and `funnel_step = -2` have different results?

Correct me if I'm wrong, but here, we have:

| funnel_step |  result |
| ------ | ---- |
| 1 | 30 |
| 2 | 15 |
| 3 | 5 |
| -1 | N.A |
| -2 | 15 |
| -3 | 10 |


Actually, this is an ideal use case for https://docs.pytest.org/en/6.2.x/parametrize.html - we can have one test with parameters like the above table, which tests all of them.

Reply:
actually they shouldn't be symmetric. When you pass in a positive value step, you're going to get all the numbers from `funnel_step` -> `max_steps`. For example, if you want `funnel_step = 1`, given the way we return results, you actually need to combine step 1, 2, 3, until max # steps since the groups are mutually exclusive

Reply:
Correct, apologies if this wasn't clear, I meant: by the way data is setup, it seems like funnel_step=2 and funnel_step=-2 give the same answer (15). Since it's not supposed to be symmetric, I'd like the test to not have symmetry just so we can protect ourselves against false positives (ex: some weird case where +2 gives answers for -2 wouldn't get caught here)

Reply:
ohh haha yeah that would be a much better test